website/src/components/Messages/MessageTableEntry.tsx

```diff
@@ -124,6 +126,7 @@ export function MessageTableEntry({ message, enabled, highlight, showAuthorBadge
         outline={highlight ? "2px solid black" : undefined}
         outlineColor={highlightColor}
         cursor={enabled ? "pointer" : undefined}
+        ref={highlight ? focusElementRef : undefined}
         overflowX="auto"
```

Comment:
have you tried with `autoFocus` on the target element? does it solve the problem?

Reply:
I didn't try.  Worth trying.  I'll try to test it out.

Reply:
I investigated, and the answer seems to be no.  React itself doesn't support autoFocus on anything except for input